### PR TITLE
docs: fix invalid links

### LINKED ISSUE #139: Fix invalid links in the Markdown files
#### Reason/Context
We have recently started validating links in the markdown files across all of the repositories across the organization. 
As I can see in the [latest check](https://github.com/asyncapi/asyncapi-php-template/actions/runs/2251640612) by the time of writing this issue, there are some invalid links in some markdown files that need fixing.

#### How to Resolve the issue?
1) If you haven't read our [contributions guide](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md), please give it a read to see how contribution works in AsyncAPI.
2) open the latest run of the [Check Markdown Links](https://github.com/asyncapi/asyncapi-php-template/actions/workflows/link-check-cron.yml) and fix them in your own fork (you can test if the check passes by running the [Check Markdown links (Weekly)](https://github.com/asyncapi/asyncapi-php-template/actions/workflows/link-check-cron.yml) in your own fork).
3) Open a Pull Request with the changes.

#### How to fix the links
* If the link is moved to a new place, edit the link to point to the appropriate place.
* If it is not supposed to be checked or it is a sample URL, add it to the ignorePatterns section of `mlc_config.json` at the root of the repo. 
* If you still don't know what to do with a specific link, ask the maintainers for help.

#### Examples
If you still have some issues, please check some example fixes in other repositories:
https://github.com/asyncapi/spec/pull/769
https://github.com/asyncapi/parser-js/pull/540
https://github.com/asyncapi/glee/pull/288
https://github.com/asyncapi/java-spring-template/pull/219

### PR DESCRIPTION
**Pull request by Maniktherana**
## Description

Fixed invalid links in markdown files across the [php template repository](https://github.com/asyncapi/asyncapi-php-template).

## Motivation and context

Fixes #139 

## How has this been tested?

Tested changes against [Check Markdown links (Weekly)](https://github.com/asyncapi/asyncapi-php-template/actions/workflows/link-check-cron.yml) in my own fork using [nektos/act](https://github.com/nektos/act).

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Contributions are **welcome** and will be fully **credited**.
 
-We accept contributions via Pull Requests on [Github](https://github.com/GA/BrokerAPI).
+We accept contributions via Pull Requests on [Github](https://github.com/pivotal-cf/brokerapi).
```
Comment (emilianozublena) [inline CONTRIBUTING.md:5]: the URL is this one: https://github.com/asyncapi/php-template make sure to updated it please :)

File: README.md [hunk 2]
```diff
@@ -84,4 +84,4 @@ The MIT License (MIT). Please see [License File](../LICENSE.md) for more informa
 [link-code-quality]: https://scrutinizer-ci.com/g/GA/BrokerAPI
 [link-downloads]: https://packagist.org/packages/GA/BrokerAPI
 [link-author]: https://github.com/emilianozublena
-[link-contributors]: ../../contributors
+[link-contributors]: https://github.com/asyncapi/asyncapi-php-template/graphs/contributors
```
Comment (emilianozublena) [inline README.md:87]: can we update the URL in here? as the name of the template (and repository) slightly changed
https://github.com/asyncapi/php-template

File: template/README.md [hunk 3]
```diff
@@ -105,4 +105,4 @@ The MIT License (MIT). Please see [License File](../LICENSE.md) for more informa
 [link-code-quality]: https://scrutinizer-ci.com/g/GA/BrokerAPI
 [link-downloads]: https://packagist.org/packages/GA/BrokerAPI
 [link-author]: https://github.com/emilianozublena
-[link-contributors]: ../../contributors
+[link-contributors]: https://github.com/asyncapi/asyncapi-php-template/graphs/contributors
```
Comment (emilianozublena) [inline template/README.md:108]: can we update the URL in here? as the name of the template (and repository) slightly changed
https://github.com/asyncapi/php-template

**Review by emilianozublena — CHANGES_REQUESTED** [review]

File: template/README.md (hunk 3 above)
Comment (Maniktherana) [reply template/README.md]: Didn't notice that! Is it recent? Just updated the links

Comment (emilianozublena) [reply template/README.md]: hi @Maniktherana yeah! we recently changed the name of the repository :)

**Review by emilianozublena — APPROVED** [review]

**Pull request merged**